packages/accounts-facebook/notice.js

```diff
@@ -1,5 +1,10 @@
-if (Package['accounts-ui'] && (Package['facebook-config-ui'] === undefined)) {
-  console.warn("Note: You're using accounts-ui and accounts-facebook, but didn't");
-  console.warn("install the configuration UI for Facebook OAuth.");
-  console.warn("You can install it with `meteor add facebook-config-ui`.");
+if (Package['accounts-ui'] && !Package.hasOwnProperty('facebook-config-ui')) {
```

Comment:
This is fine as long as we never release a package named `meteor:hasOwnProperty`. So let's, um, never do that.